lib/demo/accounts/user.ex

```diff
@@ -0,0 +1,122 @@
+defmodule Demo.Accounts.User do
+  use Ecto.Schema
+  import Ecto.Changeset
```

Comment:
Would it be better to use alias here? I really feel like explicit name-spacing is much clearer for new comers, like where to look for documentation etc.

Reply:
I am following Phoenix conventions here for `phx.gen.*`.